src/test/java/uk/gov/hmcts/reform/blobrouter/services/storage/DuplicateFileHandlerTest.java

```diff
@@ -66,11 +67,13 @@ void should_move_blob_to_rejected_container_and_create_a_new_envelope() {
 
         // then
         verify(envelopeService).createNewEnvelope(duplicate1.container, duplicate1.fileName, duplicate1.blobCreatedAt);
-        verify(envelopeService).markAsRejected(id1, DuplicateFileHandler.EVENT_MESSAGE);
+        verify(envelopeService)
+            .markAsRejected(id1, ErrorCode.ERR_ZIP_PROCESSING_FAILED, DuplicateFileHandler.EVENT_MESSAGE);
```

Comment:
This is only a recommendation, as not everyone agrees. Wherever we care about actual values of error codes, we shouldn't use the constants from application code, but use the ones defined in test code instead. Otherwise we'd be checking if the application uses the values which... it uses.